mvp.feature

```diff
@@ -1151,7 +1163,7 @@ Feature: MVP features for NVA
   # Actions from Page : My Worklist
   @1252
   Scenario: Curator expands an item in the Worklist
-    Given that a Curator views DOI Request tab
+    Given that a Curator views DOI Request tab (Part II)
     When they click Expand on an item
     Then they see the item is expanded
     And they see the Title of the Publication
```

Comment:
Is this after the expansion of the item? I do not think it should be part of this scenario.

Reply:
Do you mean that it is redundant since you can already see the title when it is not expanded?